### PR TITLE
DCS-1852 Fixing trace logging in app insights

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/utils/azureAppInsights.ts [hunk 1]
```diff
@@ -21,9 +21,7 @@ function version(): string {
 }
 
 export function initialiseAppInsights(): void {
-  // Loads .env file contents into | process.env
-  config()
-  if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
+  if (process.env.APPLICATIONINSIGHTS_CONNECTION_STRING) {
```
Comment (andrewrlee) [inline server/utils/azureAppInsights.ts:24]: this was using incorrect env var name, don't quite know how it was working previously

File: server/utils/azureAppInsights.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { config } from 'dotenv'
+import 'dotenv/config'
```
Comment (andrewrlee) [inline server/utils/azureAppInsights.ts:1]: this means we don't have to explicitly call `config()`

File: jobs/eventsJob.ts [hunk 3]
```diff
@@ -1,6 +1,5 @@
-/* eslint-disable import/first */
-/* eslint-disable import/order */
-import 'dotenv/config'
```
Comment (andrewrlee) [inline jobs/eventsJob.ts:3]: don't need as this is included as part of the line below

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**